test/e2e-go/e2e_go_olm_test.go

```diff
@@ -1,160 +0,0 @@
-// Copyright 2020 The Operator-SDK Authors
-//
```

Comment:
Why it was removed?
We should not remove this test?

Reply:
/hold

Reply:
From the description:
>Additionally, the OLM component of Go e2e tests was removed because a more thorough test is already performed by the integration tests.